### PR TITLE
Update geo-commodities.ttl

### PR DESCRIPTION
**Pull request by KellyVance**
**Additions of** 

- Concept: [Other Industrial Rock](https://vocabs.uat.gsq.digital/object?uri=http%3A//linked.data.gov.au/def/geo-commodities/other-industrial-rock)
- Concept [Other Industrial Mineral](https://vocabs.uat.gsq.digital/object?uri=http%3A//linked.data.gov.au/def/geo-commodities/other-industrial-mineral)

**Expansion of:**

- Collection [Material: Bulk Material](https://vocabs.uat.gsq.digital/object?uri=http%3A//linked.data.gov.au/def/geo-commodities/material-earthmaterial) to include:
- [Salt](https://vocabs.uat.gsq.digital/object?uri=http%3A//linked.data.gov.au/def/geo-commodities/salt)
- [Other Industrial Rock](https://vocabs.uat.gsq.digital/object?uri=http%3A//linked.data.gov.au/def/geo-commodities/other-industrial-rock)
- [Other Industrial Mineral](https://vocabs.uat.gsq.digital/object?uri=http%3A//linked.data.gov.au/def/geo-commodities/other-industrial-mineral)

### CODE REVIEW

**Review by GSQ-AI — DISMISSED** [review]
Smoke test in UAT shows that the changes do not cause issue and function as expected.

Addition of Salt, Other Industrial Rock & Mineral are logical, and they introduce requested functionality.

File: vocabularies-gsq/geo-commodities.ttl [hunk 1]
```diff
@@ -1262,6 +1262,14 @@ cdt:industrial-rock a skos:Concept ;
     skos:notation "INRK" ;
     skos:prefLabel "Industrial Rock"@en .
 
+cdt:other-industrial-rock a skos:Concept ;
+    skos:broader cdt:industrial-rock ;
+    skos:definition "An industrial rock not that is unspecified or a grouping term for rocks not available as specific commodities in a industrial rock collection. Used for specific applications where specificity is not required or may be deleterious."@en ;
```
Comment (LizDerrington) [inline vocabularies-gsq/geo-commodities.ttl:1267]: Delete 'not'

```suggestion
    skos:definition "An industrial rock that is unspecified or a grouping term for rocks not available as specific commodities in a industrial rock collection. Used for specific applications where specificity is not required or may be deleterious."@en ;
```

File: vocabularies-gsq/geo-commodities.ttl [hunk 2]
```diff
@@ -1794,6 +1802,14 @@ cdt:industrial-mineral a skos:Concept ;
     skos:notation "INMN" ;
     skos:prefLabel "Industrial Mineral"@en .
 
+cdt:other-industrial-mineral a skos:Concept ;
+    skos:broader cdt:industrial-mineral ;
+    skos:definition "An industrial mineral not that is unspecified or a grouping term for minerals not available as specific commodities in a collection, used for specific applications where specificity is not required or may be deleterious."@en ;
```
Comment (LizDerrington) [inline vocabularies-gsq/geo-commodities.ttl:1807]:
```suggestion
    skos:definition "An industrial mineral that is unspecified or a grouping term for minerals not available as specific commodities in a collection, used for specific applications where specificity is not required or may be deleterious."@en ;
```

**Review by GSQ-AI — APPROVED** [review]
Approved.

Logical and necessary.

**Review by LukeHauck — APPROVED** [review]
Reads fine to me. Tech approved too.

**Review by LizDerrington — APPROVED** [review]

**Pull request merged**